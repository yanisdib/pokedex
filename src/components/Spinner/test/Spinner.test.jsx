import { shallow } from 'enzyme';

import Spinner from '../Spinner';


describe('Spinner component', () => {
    let wrapper;

    beforeEach(() => wrapper = shallow(<Spinner />))

    it('renders correctly without crashing', () => expect(wrapper).toMatchSnapshot())
})