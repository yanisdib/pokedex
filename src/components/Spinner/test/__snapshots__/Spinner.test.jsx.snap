// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders correctly without crashing 1`] = `ShallowWrapper {}`;
